Command/RemoveCacheCommand.php

```diff
@@ -20,56 +21,89 @@
 
 class RemoveCacheCommand extends Command
 {
-    /* @var CacheManager cacheManager */
-    private $cacheManager;
+    use CacheCommandTrait;
 
-    public function __construct(CacheManager $cacheManager)
+    public function __construct(CacheManager $cacheManager, FilterManager $filterManager)
     {
-        $this->cacheManager = $cacheManager;
+        parent::__construct();
 
-        parent::__construct('liip:imagine:cache:remove');
+        $this->cacheManager = $cacheManager;
+        $this->filterManager = $filterManager;
     }
 
-    protected function configure()
+    protected function configure(): void
     {
         $this
-            ->setDescription('Remove cache for given paths and set of filters.')
-            ->addArgument('paths', InputArgument::OPTIONAL | InputArgument::IS_ARRAY, 'Image paths')
-            ->addOption(
-                'filters',
-                'f',
-                InputOption::VALUE_OPTIONAL | InputOption::VALUE_IS_ARRAY,
-                'Filters list'
-            )
+            ->setName('liip:imagine:cache:remove')
```

Comment:
why move the name from constructor to here? afaik the symfony 4 way is a static protected property for the name, which can avoid instantiating the command in some situations.

Reply:
No idea, I added the protected property in both commands